_docs/README.md

```diff
@@ -164,7 +176,11 @@ variables or dependencies will be available.
 ```yaml
 variables:
   - name: <NAME>
-    prompt: <PROMPT>
+    description: <DESCRIPTION>
+    type: <TYPE>
+    options:
+      - <CHOICE>
+      - <CHOICE>
     default: <DEFAULT>
```

Comment:
Maybe a case can still be made for both `prompt` and `description`. For example, up above, you have the following example:

```
ShowLogo
   Should the website show the logo?

   Enter a [type: bool]: true
```

But is "Should the website show the logo?" really a `description`? That seems more like a `prompt` to me. What if the rendering looked something like this?

```
ShowLogo
   Should the website show the logo?

   Showing the logo helps promote the website but also takes more bandwidth.

   Enter a [type: bool]: true
```

The top text would be the `prompt` and the middle text the `description`. Also, I believe you're using colors in the terminal output now? You could make the description grey to further de-emphasize it.